bigframes/operations/__init__.py

```diff
@@ -664,6 +676,46 @@ def output_type(self, *input_types: dtypes.ExpressionType) -> dtypes.ExpressionT
 
 clip_op = ClipOp()
 
+
+class SwitchOp(NaryOp):
```

Comment:
Could we name this `CaseWhenOp` to match SQL https://cloud.google.com/bigquery/docs/reference/standard-sql/conditional_expressions ?

TIL: This is a supported op name in pandas now too: https://pandas.pydata.org/docs/reference/api/pandas.Series.case_when.html (learned via https://stackoverflow.com/a/73619829/101923)

Reply:
renamed